python_ta/__main__.py

```diff
@@ -20,17 +21,42 @@
     "filenames", nargs=-1, type=click.Path(exists=True, dir_okay=True, resolve_path=True)
 )
 @click.option("--exit-zero", is_flag=True, help="Always return with status code 0", default=False)
-def main(config: Optional[str], errors_only: bool, filenames: List[str], exit_zero: bool) -> None:
+@click.option(
+    "-g",
+    "--generate-config",
+    is_flag=True,
+    help="Print out default PythonTA configuration file",
+    default=False,
+)
+@click.option(
+    "--output-format",
+    help="Specify the format of output report",
+    default="python_ta.reporters.HTMLReporter",
+)
+def main(
+    config: Optional[str],
+    errors_only: bool,
+    filenames: List[str],
+    exit_zero: bool,
+    generate_config: bool,
+    output_format: str,
+) -> None:
     """A code checking tool for teaching Python.
 
     FILENAMES can be a string of a directory, or file to check (`.py` extension optional) or
     a list of strings of directories or files.
     """
     # `config` is None if `-c` flag is not set
+    if generate_config:
+        pylintrc_location = os.path.join(os.path.dirname(__file__), ".pylintrc")
+        with open(pylintrc_location, "r") as f:
+            contents = f.read()
+            print(contents)
+            sys.exit(0)
+
     checker = check_errors if errors_only else check_all
     paths = [click.format_filename(fn) for fn in filenames]
-    reporter = checker(module_name=paths, config=config)
-
+    reporter = checker(module_name=paths, config={"output-format": output_format})
```

Comment:
Ah I missed this the first time, but I realize this option currently conflicts with the `config` option. To make things simple, for now please make two changes:

1. Use the previous code (`config=config`) if `config` is not `None`, otherwise use this new code. In other words, the `--config` argument takes precedence over `--output-format`.
2. Add to the `help` text for this option to say that "This option is ignored if a `--config` argument is specified"